### ISSUE TITLE
figure out why I can't get the boostrap-icons to be included

### ISSUE THREAD

**Issue by bennyvasquez** [AlmaLinux/almalinux.org#633]
I want to be able to use more and different boostrap icons than have been included in our hugo install. My googling lead me to just including the CSS files by default our installs. I cannot get these files to be rendered correctly for some reason, and am assuming it's a lack of hugo/boostrap knowledge somewhere. 

This is what I did to try to get them to work: 

[commit one](https://github.com/AlmaLinux/almalinux.org/pull/621/commits/202de896244dbe58ab792e246f83f702a56ac2b0) [commit two](https://github.com/AlmaLinux/almalinux.org/pull/621/commits/e1d8dfdcbf3fa14a4178fc13411b60fc76c64b01) 

I've included the bi-building-fill-check icon in the sub-title of the software-certification and hardware-certification pages, so you can use those as your testing when trying to get it working, if you'd like.

**Comment by pastalian** [comment]
I think the cause of issue is the outdated bootstrap-icons. 

https://github.com/AlmaLinux/almalinux.org/blob/1376f835e51418890a806e27ae69ff23a014e8f2/layouts/partials/common/head.html#L33

```diff

-    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.3.0/font/bootstrap-icons.css">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.11.3/font/bootstrap-icons.css">
```

Updating to 1.11.3 should fix it.

**Comment by bennyvasquez** [comment]
you're ABSOLUTELY correct. I updated it in a different PR and it fixed the problem. As soon as it's merged in I'll close this one. Thank you so much @pastalian!
